restapi/src/main/java/io/stargate/web/docsapi/exception/XNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package io.stargate.web.docsapi.exception;
```

Comment:
Please add the standard copyright header, e.g. copy from `DocCollection`

In fact these headers appear to be missing in a lot of Docs API node... I'm not sure why... @dougwettlaufer the standard header is appropriate for all of the Docs API code, right?

Reply:
Done. But just for this new class, we should have a proper clean up task to handle other missing class and enforce a IntelliJ rule that can be shared in the repo, so that anybody working can get it directly without much hassle.